### PR TITLE
Well well well...

### PR DESCRIPTION
**Pull request by Fortifier42**

### CODE REVIEW

File: plugin/src/main/java/net/aufdemrand/denizen/objects/dEntity.java [hunk 1]
```diff
@@ -2769,6 +2783,20 @@ public void adjust(Mechanism mechanism) {
...
+        // @name collidable
+        // @input Element(Boolean)
+        // @description
+        // Sets whether the entity can be collided with.
+        // Must be a living entity.
+        // @tags
+        // <e@entity.collidable>
+        // -->
+        if (mechanism.matches("collidable") && isLivingEntity() && mechanism.requireBoolean()) {
+            getLivingEntity().setCanPickupItems(value.asBoolean());
```
Comment (mcmonkey4eva) [inline plugin/src/main/java/net/aufdemrand/denizen/objects/dEntity.java]: Ehm, wot?

Comment (Fortifier42) [inline plugin/src/main/java/net/aufdemrand/denizen/objects/dEntity.java]: oops

**Comment by Morphan1** [comment]
.

**Comment by Morphan1** [comment]
@mcmonkey4eva anything stopping this from being pulled?

**Comment by mcmonkey4eva** [comment]
iirc: it don't work right and was untested for the parts that might maybe work right.

**Comment by mcmonkey4eva** [comment]
Closing as this seems dead.

**Pull request closed without merging**